edb/schema/roles.py

```diff
@@ -49,6 +49,13 @@ class Role(so.GlobalObject, so.InheritingObject,
         allow_ddl_set=True,
         inheritable=False)
 
+    password_hash = so.SchemaField(
```

Comment:
Why not just allow setting hash to the `password` field directly?

Reply:
Do you mean detecting prefix in the password? I think it's weird. While it's unlikely that your password would start with `SCRUM-SHA-256$`, but still matching on that text sounds like a bad design. Or this isn't what you propose?

Reply:
OK, I wanted to see if you considered that option.  A separate DDL field is OK too.